google/cloud/bigtable/table_admin_test.cc

```diff
@@ -17,10 +17,14 @@
 #include "google/cloud/bigtable/admin/bigtable_table_admin_options.h"
 #include "google/cloud/bigtable/admin/mocks/mock_bigtable_table_admin_connection.h"
 #include "google/cloud/bigtable/testing/mock_policies.h"
+#include "google/cloud/internal/background_threads_impl.h"
 #include "google/cloud/internal/time_utils.h"
-#include "google/cloud/testing_util/async_sequencer.h"
-#include "google/cloud/testing_util/mock_completion_queue_impl.h"
+#include "google/cloud/testing_util/is_proto_equal.h"
+#include "google/cloud/testing_util/status_matchers.h"
+#include "google/bigtable/admin/v2/bigtable_table_admin.pb.h"
```

Comment:
this include shouldn't be here. I will wait for review comments before making the change though.